src/components/Editor/Tabs.js

```diff
@@ -482,7 +487,8 @@ export default connect(
       searchTabs: getSearchTabs(state),
       sourceTabs: getSourcesForTabs(state),
       activeSearch: getActiveSearch(state),
-      searchOn: getActiveSearch(state) === "source"
+      searchOn: getActiveSearch(state) === "source",
+      sourceMetaData: getSourceMetaData(state, 0)
```

Comment:
@jasonLaster I was actually having a problem with how to fetch the `sourceId` for `getSourceMetaData`